tests/integration/controller/manifest/keychain_test.go

```diff
@@ -1,87 +0,0 @@
-package manifest_test
-
-import (
-	"os"
-	"path/filepath"
-
-	apicorev1 "k8s.io/api/core/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/yaml"
-
-	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
-	"github.com/kyma-project/lifecycle-manager/pkg/ocmextensions"
-	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
-	"github.com/kyma-project/lifecycle-manager/tests/integration"
-
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe(
-	"test authnKeyChain", func() {
-		It(
-			"should fetch authnKeyChain from secret correctly", FlakeAttempts(5), func() {
-				By("install secret")
-				const CredSecretLabelValue = "test-operator"
-				Eventually(installCredSecret(kcpClient, CredSecretLabelValue), standardTimeout,
-					standardInterval).Should(Succeed())
-				const repo = "test.registry.io"
-				imageSpecWithCredSelect := CreateOCIImageSpecWithCredSelect("imageName", repo,
-					"digest", CredSecretLabelValue)
-				keychain, err := ocmextensions.GetAuthnKeychain(ctx,
-					imageSpecWithCredSelect.CredSecretSelector, kcpClient)
-				Expect(err).ToNot(HaveOccurred())
-				dig := &TestRegistry{target: repo, registry: repo}
-				authenticator, err := keychain.Resolve(dig)
-				Expect(err).ToNot(HaveOccurred())
-				authConfig, err := authenticator.Authorization()
-				Expect(err).ToNot(HaveOccurred())
-				Expect(authConfig.Username).To(Equal("test_user"))
```

Comment:
Is there a replacement for this test? While it uses selector, it confirms that the resulting authConfig is correct.

Reply:
Good catch! Seems I removed this accidentally. Will introduce a replacement 👌